api/src/main/java/com/epam/pipeline/manager/git/ApiBuilder.java

```diff
@@ -47,17 +47,19 @@
     private final int readTimeout;
     private final String apiHost;
     private final String adminToken;
+    private final String bearerToken;
     private Class<T> apiClientClass;
     private String dateFormat;
 
     public ApiBuilder(final Class<T> apiClientClass, final String apiHost,
-                      final String adminToken, final String dateFormat) {
+                      final String adminToken, final String bearerToken, final String dateFormat) {
         this.apiClientClass = apiClientClass;
         this.dateFormat = dateFormat;
         this.connectTimeout = TIMEOUT;
         this.readTimeout = TIMEOUT;
         this.apiHost = apiHost;
         this.adminToken = adminToken;
+        this.bearerToken = bearerToken;
```

Comment:
I don't think that we'll ever need two tokens at the same time. Can we switch to `String authHeaderName` and `String token` fields instead of passing two tokens?